packages/core/src/components/list-button/list-button.scss

```
@@ -1,9 +1,9 @@
 @use "~dso-toolkit/src/variables/units";
-@use "~dso-toolkit/src/components/input-number";
 @use "~dso-toolkit/src/components/list-button";
 @use "~dso-toolkit/src/components/button";
 @use "~dso-toolkit/src/components/selectable";
 @use "~dso-toolkit/src/components/form-control";
+@use "~dso-toolkit/src/components/form/form.mixins" as form-mixins;
```

De stylesheets achter de componenten moet je als een module zien (en daarmee bedoel ik niet Sass modules - we gebruiken ze er wel voor).

Dat Form gebruikt maakt van mixins is voor List Button niet relevant. Form publiceert wat nodig is via de `_index.scss` file.

Een ander component kan via die exports importen (usen) wat nodig is. Vergelijk het met een mini-public API van een component.

```suggestion
@use "~dso-toolkit/src/components/form" as form;

.iets {
  @include form.input-number();
}
```

Op deze manier hebben alle modules grip op wat ze beschikbaar stellen. En realiseren wij ons (als maintainers) wat we van andere componenten nodig hebben: Als een ding (variable, mixin, function) wordt gevraagd van een component maar niet beschikbaar wordt gesteld, is dat een signaal om bij stil te staan.

---

Er is vast een reden waarom er geen _index.scss bestaat in form. Maar bovenstaande suggestie werkt niet. Zal ik dan een _index.scss maken? @tfrijsewijk